bigtable/docs/snippets.py

```diff
@@ -147,20 +149,19 @@ def test_bigtable_create_additional_cluster():
     cluster.delete()
 
 
-def test_bigtable_create_app_profile():
+def test_bigtable_create_reload_delete_app_profile():
+    import re
     # [START bigtable_create_app_profile]
     from google.cloud.bigtable import Client
 
     client = Client(admin=True)
     instance = client.instance(INSTANCE_ID)
 
-    app_profile_id = "app-prof-" + unique_resource_id("-")
     description = "routing policy-multy"
-    routing_policy_type = enums.RoutingPolicyType.ANY
 
     app_profile = instance.app_profile(
-        app_profile_id=app_profile_id,
-        routing_policy_type=routing_policy_type,
+        app_profile_id=APP_PROFILE_ID,
+        routing_policy_type=ROUTING_POLICY_TYPE,
         description=description,
```

Comment:
now that the others are named constants this seems inconsistent. Also, is their guidance from sample authoring to start using hidden constants? I only ask as having the values initialized in the sample helps the user understand what that value looks like.

Are `APP_PROFILE_ID` and `ROUTING_POLICY_TYPE` well understood vars? The other two constants are so I think they are fine (`CLUSTER_ID` and `INSTANCE_ID`)

Reply:
App profile id is well defined. Routing policy type might be good to include in the sample since it is an enum, so you can show how to access it

Reply:
Implemented review. This was good suggestion.